frontend/src/components/DateSelector/DateSelector.js

```diff
@@ -0,0 +1,75 @@
+import { useSelector } from "react-redux";
+import Container from "react-bootstrap/Container";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import Dropdown from "react-bootstrap/Dropdown";
+import { useState } from "react";
+
+function DateSelector() {
+  const [month, setMonth] = useState(
```

Comment:
we'll want to store these as numbers in the state not the names since that's what we'll pass to the backend.

you can have an array of all the month name strings. but this stores the month index plus 1 since months don't start at 0.

so the eventKey for months will be the number 1 = January. the initial value should be the current month number